internal/commands/list.go

```diff
@@ -51,14 +55,13 @@ func listCmd(dockerCli command.Cli) *cobra.Command {
 			return runList(dockerCli, opts)
 		},
 	}
-	cmd.Flags().StringVar(&opts.targetContext, "target-context", "", "List installations on this context")
 
 	return cmd
 }
 
 func runList(dockerCli command.Cli, opts listOptions) error {
-	targetContext := getTargetContext(opts.targetContext, dockerCli.CurrentContext())
-	installations, err := getInstallations(targetContext, config.Dir())
+	opts.SetDefaultTargetContext(dockerCli)
```

Comment:
Can't we simplify it with just calling ?
```go
targetContext := getTargetContext(dockerCli.CurrentContext())
```
And by the way simplify `getTargetContext` func too?